src/Api/TMDB/tmdbService.tsx

```diff
@@ -0,0 +1,91 @@
+export const fetchMovieGenres = async () => {
```

Comment:
why a tsx file for this? Services should be pure ts/js files . Are you using anything react specific or anything that needs jsx ? if not, please move it to ts.

Reply:
Renamed the file from tsx to ts as there no use of react in the file